rpm/lipstick2vnc.spec

```diff
@@ -66,6 +66,9 @@ fi
 %files
 %defattr(-,root,root,-)
 %attr(2755, root, privileged) %{_bindir}/%{name}
+/usr/share/lipstick2vnc/cursor_empty.png
```

Comment:
Should be %{_datadir} instead of /usr/share but I can adjust that. Could have a bunch of other cleanups as well.